src/Makefile

```diff
@@ -107,25 +102,23 @@ $(BLIB)/libveo.a: $(VHLIB_OBJS) | $$(@D)/
 	$(AR) rv $@ $^
 
 $(BLIBEX)/aveorun: $(BVE)/aveorun.o | $$(@D)/
-	if [ -z $(PREF) ]; then \
-		$(NFORT) $(NFORTFLAGS) -v -cxxlib -fopenmp -o $@ $^ -L. \
-			-L$(BLIB) -L$(BVELIB) -Wl,-rpath,/opt/nec/ve/lib \
-			$(NLDFLAGS) -laveoVE -lveio -lpthread;\
-	else \
-		$(NFORT) $(NFORTFLAGS) -v -cxxlib -fopenmp -o $@ $^ -L. \
-			-L$(BLIB) -L$(BVELIB) -Wl,-rpath,/opt/nec/ve/lib \
-			-laveoVE -lveio -lpthread;\
-	fi
+	$(NFORT) $(NFORTFLAGS) -v -cxxlib -fopenmp -o $@ $^ \
+		$(NLDFLAGS) -laveoVE -lveio -lpthread
 
 $(BLIBEX)/aveorun-ftrace: $(BVE)/aveorun-ftrace.o | $$(@D)/
-	$(NFORT) $(NFORTFLAGS) -ftrace -cxxlib -fopenmp -o $@ $^ $(NLDFLAGS) -L. \
-		-L$(BLIB) -laveoVE -lveio -lpthread
+	$(NFORT) $(NFORTFLAGS) -ftrace -cxxlib -fopenmp -o $@ $^ \
+		$(NLDFLAGS) -laveoVE -lveio -lpthread
 
 $(BLIBEX)/relink_aveorun: ../scripts/relink_aveorun.in
 	sed -e "s,@libexecdir@,$(DEST)/libexec,g" -e "s,@velibdir@,$(BUILD)/lib,g" \
 		< $< > $@
 	chmod 755 $@
 
+$(BLIBEX)/mk_veorun_static: ../scripts/relink_aveorun.in
```

Comment:
Do we need twice the same script? IMO relink_aveorun is a more appropriate name and we should keep it and symlink the old name for compatibility.

Reply:
We pushed the patch to add relink_aveorun to aveorun package.
We made mk_veorun_static a symbolic link.